src/dotnet/APIView/APIView/MethodAPIV.cs

```diff
@@ -0,0 +1,123 @@
+﻿using Microsoft.CodeAnalysis;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+
+namespace APIView
+{
+    /// <summary>
+    /// Class representing a C# method. Each method includes a name, return type, attributes, 
+    /// modifiers, type parameters, and parameters.
+    /// 
+    /// Method is an immutable, thread-safe type.
+    /// </summary>
+    public class MethodAPIV
+    {
+        public IMethodSymbol Symbol { get; }
+
+        public string Name { get; }
+        public string ReturnType { get; }
```

Comment:
Add TypeReference type that we talked about (for now with Name and FullName properties)